packages/hash/integration/src/mockData/index.ts

```diff
@@ -69,30 +69,30 @@ void (async () => {
   const results = new Map<string, Entity>();
 
   const requiredBlockTypes = [
-    "Divider",
-    "Embed",
-    "Image",
-    "Table",
-    "Code",
-    "Video",
-    "Header",
+    ["Divider", "https://blockprotocol.org/blocks/@hash/divider"],
+    ["Embed", "https://blockprotocol.org/blocks/@hash/embed"],
+    ["Image", "https://blockprotocol.org/blocks/@hash/image"],
+    ["Table", "https://blockprotocol.org/blocks/@hash/table"],
+    ["Code", "https://blockprotocol.org/blocks/@hash/code"],
+    ["Video", "https://blockprotocol.org/blocks/@hash/video"],
+    ["Header", "https://blockprotocol.org/blocks/@hash/header"],
   ] as const;
 
   const requiredOtherTypes = ["Company", "Location", "Person"] as const;
   // create the types we'll need below so we can assign their ids to entities
   const newTypeIds: Record<
-    typeof requiredBlockTypes[number] | typeof requiredOtherTypes[number],
+    typeof requiredBlockTypes[number][0] | typeof requiredOtherTypes[number],
```

Comment:
Optional: Change `requiredBlockTypes` to `[{ name, componentId }]` or a like to avoid `[0]`. Code with index-based tuples is a bit harder to read.